libvast/CMakeLists.txt

```diff
@@ -318,8 +318,9 @@ target_link_libraries(libvast PUBLIC ${ARROW_LIBRARY})
 dependency_summary("Apache Arrow" ${ARROW_LIBRARY} "Dependencies")
 
 # Link against Boost.
-target_link_libraries(libvast PRIVATE Boost::headers Boost::filesystem)
-target_link_libraries(libvast_builtins PRIVATE Boost::headers Boost::filesystem)
+target_link_libraries(libvast PUBLIC Boost::headers Boost::filesystem)
```

Comment:
Why did you change it to public? Afaik we aren't using boost in the interface.

Reply:
"you" = @dominiklohmann 

Just to make sure it pops up in your feed. 🙂

Reply:
That has to be because it's used in a builtin. We link the builtins against plugin unit tests so they can be used in them, and plugin unit tests may be built against an installed libvast. So that's why it's got to be public.

Reply:
Thanks for the explanation. I think I have an idea for what we can do about that, but I guess it's low priority.